### PR TITLE
Barracuda filter option

### PR DESCRIPTION
**Pull request by vikibaarathi**
Moved DSHOT section to the filter section to keep things aligned and standardised.

### CODE REVIEW

**Comment by vikibaarathi** [comment]
I should actually move the warning into this section also right?

**Comment by limonspb** [comment]
> I should actually move the warning into this section also right?

Warnings can stay where they are now. They don't work properly if moved in options, thats something i have to work on.

Please remove extra while spaces in the option, and we good to go.
image

**Review by haslinghuis — DISMISSED** [review]
Note Dshot 600 will not enable with dshot_bidir enabled for F4 | G4 MCU's

**Comment by limonspb** [comment]
> Note Dshot 600 will not enable with dshot_bidir enabled for F4 | G4 MCU's

ok i just tested, it goes to dshot 300 when i set it to 600 on F405.
So i guess trying to set it to 600 is not a problem?

**Comment by limonspb** [comment]
> I should actually move the warning into this section also right?

sorry there is one empty line left there in the option lol

File: presets/4.4/tune/barracuda_race_tune.txt [hunk 1]
```diff
@@ -30,6 +30,10 @@
     #$ INCLUDE: presets/4.3/filters/defaults.txt
     set simplified_gyro_filter_multiplier = 140
     set gyro_lpf2_static_hz = 0
+    set dyn_notch_count = 1
+    set dyn_notch_q = 500
+    set dshot_bidir = ON
+    set motor_pwm_protocol = DSHOT600
```
Comment (haslinghuis) [inline presets/4.4/tune/barracuda_race_tune.txt:37]:
```suggestion
```

File: presets/4.4/tune/barracuda_race_tune.txt [hunk 2]
```diff
@@ -42,11 +46,6 @@ set simplified_i_gain = 110
 set simplified_dmax_gain = 0
 set simplified_feedforward_gain = 80
```
Comment (haslinghuis) [inline presets/4.4/tune/barracuda_race_tune.txt:48]:
```suggestion
```

**Review by limonspb — APPROVED** [review]

**Comment by vikibaarathi** [comment]
THank you for helping me clean up @limonspb. I'll check better on the next one.

**Comment by limonspb** [comment]
> THank you for helping me clean up @limonspb. I'll check better on the next one.

no worries Sir

**Pull request merged**